README.md

```diff
@@ -22,6 +22,13 @@ The policy demonstrates that it's possible to perform validation and make use of
 [Kubewarden's host capabilities](https://docs.kubewarden.io/reference/spec/host-capabilities/intro-host-capabilities)
 during policy evaluation.
 
+## Project Structure
+The project is organized into the following components:
+
+- **`js/`**: The core TypeScript SDK.
+- **`js/demo-policy/`**: A sample policy demonstrating the SDKs capabilities.
+- **`javy-plugin-kubewarden/`**: Custom Javy plugin for JavaScript to WebAssembly compilation
```

Comment:
Can we move the `js/demo-policy/` out of the `js` directory? Something like: 
- `js`
- `demo-policy`
- `javy-plugin-kubewarden`